devito/operators.py

```diff
@@ -0,0 +1,40 @@
+from jit_manager import JitManager
+from propagator import Propagator
+import os
+
+
+class Operator(object):
```

Comment:
It seems like it it not used anywhere, probably a reminiscence of the old one we din't remove. I'll remove it.